### PR TITLE
38 many language versions

### PR DESCRIPTION
**Pull request by joanna-bukowska**

### CODE REVIEW

File: DrinkManagerWeb/Startup.cs [hunk 1]
```diff
@@ -34,6 +36,18 @@ public void ConfigureServices(IServiceCollection services)
                 })
                 .AddEntityFrameworkStores<DrinkAppContext>();
 
+            services.AddLocalization();
+            services.Configure<RequestLocalizationOptions>(options =>
+            {
+                options.SetDefaultCulture("en-GB");
+                options.AddSupportedCultures("en-GB","pl-PL");
+                options.FallBackToParentUICultures = true;
+
```
Comment (wojciechkapko) [inline DrinkManagerWeb/Startup.cs:51]: Here i had to add `options.AddSupportedUICultures("en-GB", "pl-PL");` in order to make it work on my machine.

It was probably due to the fact that my PC's default language culture is not Polish.

File: DrinkManagerWeb/Startup.cs [hunk 2]
```diff
@@ -53,8 +67,16 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IDrinkSearchService, DrinkSearchService>();
             services.AddScoped<IFavouriteRepository, FavouriteRepository>();
             services.AddScoped<IReviewRepository, ReviewRepository>();
+            services
+                .AddRazorPages()
+                .AddViewLocalization();
+            services.AddMvc()
```
Comment (wojciechkapko) [inline DrinkManagerWeb/Startup.cs]: AddMvc method was used for .Net Core 2.0-2.2 in 3.1 we should chain AddDataAnnotationsLocalization after AddControllersWithViews.

File: DrinkManagerWeb/Startup.cs (hunk 1 above)
Comment (joanna-bukowska) [reply DrinkManagerWeb/Startup.cs:51]: Added

File: DrinkManagerWeb/Startup.cs (hunk 2 above)
Comment (joanna-bukowska) [reply DrinkManagerWeb/Startup.cs]: Changed

**Review by marek-wilk — APPROVED** [review]

**Review by wojciechkapko — APPROVED** [review]

**Pull request merged**